js/stdlib/ts/CBR.ts

```diff
@@ -16,7 +16,7 @@ export const bigNumberToNumber = (x: any) =>
 export type CBR_Null = null;
 export type CBR_Bool = boolean;
 export type CBR_UInt = BigNumber;
-export type CBR_Bytes = string;
+export type CBR_Bytes = Uint8Array | string;
```

Comment:
This needs to be documented in https://docs.reach.sh/frontend/#ref-frontends-js-types

Reply:
Make sure you say something about when it is one versus the other